### PR TITLE
Fix type error

### PR DESCRIPTION
**Pull request by microshine**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: pkcs11js/pkcs11js.d.ts [hunk 1]
```diff
@@ -1173,11 +1173,11 @@ declare module "pkcs11js" {
     const CKM_RSA_PKCS_TPM_1_1: number;
     const CKM_RSA_PKCS_OAEP_TPM_1_1: number;
     const CKM_VENDOR_DEFINED: number;
-
+
```
Comment (andy-ms) [inline pkcs11js/pkcs11js.d.ts]: Unnecessary whitespace

Comment (microshine) [inline pkcs11js/pkcs11js.d.ts]: @andy-ms I fixed it. I've got one question about TSD. New version of TS is available v2.0.3. When is TSD going to use lib.d.ts from the newest version of TS. Cause I've got TS error with old my projects which used v1.8

Comment (andy-ms) [inline pkcs11js/pkcs11js.d.ts]: @microshine: I don't know; you could ask at [Definitelytyped/tsd](https://github.com/DefinitelyTyped/tsd). (Or you could always transition to `@types`! See the [blog post](https://blogs.msdn.microsoft.com/typescript/2016/09/22/announcing-typescript-2-0/))

Comment (microshine) [inline pkcs11js/pkcs11js.d.ts]: @andy-ms Thank you
